Makefile

```diff
@@ -46,7 +46,7 @@ RELEASE_OUT ?=
 TELEPORT_TAG = 3.2.17
 # TELEPORT_REPOTAG adapts TELEPORT_TAG to the teleport tagging scheme
 TELEPORT_REPOTAG := v$(TELEPORT_TAG)
-PLANET_TAG := 8.0.4-$(K8S_VER_SUFFIX)
+PLANET_TAG := 8.0.6-$(K8S_VER_SUFFIX)
```

Comment:
Looks like this is also introducing some unrelated changes from https://github.com/gravitational/planet/releases/tag/8.0.5-11912 / https://github.com/gravitational/planet/pull/859.  I don't anticipate any risk here, but I also haven't reviewed that code myself.